domain/src/test/java/org/oppia/android/domain/oppialogger/OppiaLoggerTest.kt

```diff
@@ -120,6 +126,28 @@ class OppiaLoggerTest {
     assertThat(eventContext.revisionCardContext.subTopicId).isEqualTo(TEST_SUB_TOPIC_ID)
   }
 
+  @Test
+  fun testController_logException_nonFatal_logsToRemoteService() {
+    val exceptionThrown = Exception("TEST MESSAGE", Throwable())
+    oppiaLogger.logNonFatalException(exceptionThrown, TEST_TIMESTAMP_IN_MILLIS_ONE)
+
+    val exceptionLogged = fakeExceptionLogger.getMostRecentException()
+
+    assertThat(exceptionLogged).isEqualTo(exceptionThrown)
+  }
+
+  @Test
+  fun testController_logFatalException_logsToRemoteService() {
+    val exceptionThrown = Exception("TEST MESSAGE", Throwable())
+    oppiaLogger.logFatalException(
+      exceptionThrown, TEST_TIMESTAMP_IN_MILLIS_ONE
+    )
+
+    val exceptionLogged = fakeExceptionLogger.getMostRecentException()
+
+    assertThat(exceptionLogged).isEqualTo(exceptionThrown)
+  }
+
```

Comment:
I think there are more tests that can be added here for the exceptions like when there is no network connectivity and they are stored into the cache store. Refer to the `ExceptionControllerTests` for these.

Reply:
The other test cases required some ```exceptionController``` functions. Should I put it like that? for ex: 
![Capture](https://user-images.githubusercontent.com/69083588/98712900-e9df3a80-23ac-11eb-9ac2-7dcd5decad9a.PNG)
 The ```getExceptionLogStore``` function is in ```exceptionsController```. I tried adding the function in ```oppiaLogger``` just to check then the ```LiveData()``` had error.

Reply:
I think you can use these functions in the tests without adding them to the OppiaLogger.